package.json

```diff
@@ -3,14 +3,15 @@
   "version": "0.0.1",
   "description": "Windbreaker service utilities",
   "scripts": {
-    "lint": "eslint .",
-    "docker:start": "docker-compose up -d",
+    "docker:build": "docker-compose build",
+    "docker:start": "docker-compose up rediscluster rabbitmq",
     "docker:stop": "docker-compose down",
-    "pretest": "npm run lint",
```

Comment:
I removed this `pretest` hook because I wanted to allow for all dev to happen within docker containers.